src/plugins/plugin.ts

```diff
@@ -14,7 +14,7 @@ export { IBlock, StringOrTag };
  */
 export interface IFile {
     path: string;
-    read: () => string;
+    read(): string;
```

Comment:
Nothing! They are equivalent. This is more like an instance member whereas the former is like an object property. Subjective!